Monotype/c2pa-font-ux-js
# CP2A-170: New repo for C2PA font web components

# Issue

- [C2PA-170](https://monotype.atlassian.net/browse/C2PA-170)

# Checklist


# Notes for Reviewers

Basic repository documents. Please review them in their entirety.





# Verification Instructions

I modified the usual language to refer to GitHub Issues (since this is a public repo) instead of JIRA.

Regarding the repo settings, I can't seem to see them - I have a message in to the Help Desk to see about getting admin privileges, but it's also possible some of these settings work differently on public repos (vs. the private one with which we are most familiar).


















> Actively maintained by the @Monotype/driverpdldev team.

[C2PA-170]: https://monotype.atlassian.net/browse/C2PA-170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
